web/src/App.jsx

```diff
@@ -18,8 +18,10 @@ import PropTypes from "prop-types";
 export const usePersistentStore = create(
     persist(
         (set) => ({
-            selectedSource: 0,
+            selectedSource: null,
             setSelectedSource: (selected) => {
+                console.log("SELECTED:");
+                console.log(selected);
```

Comment:
```suggestion
```

Reply:
Good catch, this seems to have made it in due to me switching branches after testing #788